app/ledger.js

```diff
@@ -354,8 +344,14 @@ eventStore.addChangeListener(() => {
  * module initialization
  */
 
-var initialize = () => {
-  enable(getSetting(settings.PAYMENTS_ENABLED))
+var initialize = (onoff) => {
+  enable(onoff)
```

Comment:
is there any reason to keep `enable` as a module entry point? when would it get called independently of initialize?